src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,1503 @@
+                <td>
+                <p>
+                  <ix:nonNumeric contextRef="EntityContactInfo_CY" name="uk-bus:AddressLine1">{{$roa.address_line_1}}</ix:nonNumeric>
+                  </p>
+                  <p>
+                  <ix:nonNumeric contextRef="EntityContactInfo_CY" name="uk-bus:AddressLine2">{{$roa.address_line_2}}</ix:nonNumeric>
+                  </p>
+                  <p>
+                  <ix:nonNumeric contextRef="EntityContactInfo_CY" name="uk-bus:CountyRegion">{{$roa.county}}</ix:nonNumeric>
+                  </p>
+                  <p>
+                  <ix:nonNumeric contextRef="EntityContactInfo_CY" name="uk-bus:PostalCodeZip">{{$roa.postcode}}</ix:nonNumeric>
+                  </p>  
+                </td>
+              </tr>
+              <tr>
+                <td> </td>
+              </tr>
+              <tr>
+                <td> </td>
+              </tr>
+              <tr>
+                <td>
+                    <b>Company Registration Number:</b>
+                </td>
+                <td> </td>
+                <td>
+                  <ix:nonNumeric contextRef="CY" name="uk-bus:UKCompaniesHouseRegisteredNumber">{{$companyNumber}}</ix:nonNumeric>
+                        ({{$company.ROA.jurisdiction }}) 
+                </td>
+              </tr>
+              <tr>
+                <td> </td>
+             </tr>
+           </tbody>
+          </table>
+        </div>
+        
+        <!-- BALANCE SHEET -->
+        
+        {{$currentPeriod := .small_full_account.period.current_period_start_on}}
+        {{$previousPeriod := .small_full_account.period.previous_period_start_on}}
+        <div class="page">
+          <div class="header">
+            <h1 class="text--center">
+              <ix:nonNumeric contextRef="CY" name="uk-bus:EntityCurrentLegalOrRegisteredName">{{$companyName}}</ix:nonNumeric>
+            </h1>
+          </div>
+          <h2 class="text--center">Balance sheet</h2>
+          <p class="text--center">
+            <strong>As at
+              <ix:nonNumeric format="ixt2:datedaymonthyearen" contextRef="CY_END" name="uk-bus:BalanceSheetDate">{{convertDateToDisplayFormat $current_period_end_on}}</ix:nonNumeric>
+            </strong>
+          </p>
+          <table cellspacing="10">
+          <thead>
+              <tr>
+                <th class="table-heading"></th>
+                <th class="table-heading"> <br />Notes </th>
+            
+                {{if $currentPeriod}}
+                  <th class="table-heading">
+                     {{$balanceSheet.current_period_date_formatted}}
+                  </th>
+                {{end}}
+                {{if $previousPeriod}}
+                  <th class="table-heading">
+                   {{$balanceSheet.previous_period_date_formatted}}
+                  </th>
+                {{end}}
+              </tr>
+
+              <tr>
+                <th class="table-heading"></th>
+                <th cla
```

Comment:
`contextRef="Director40_CY"` will need to be `contextRef="Director1_CY"` as it relates
`<ix:nonNumeric contextRef="Director1_CY" name="core:DirectorSigningFinancialStatements" format="ixt2:nocontent"></ix:nonNumeric>`